tvb_nest/config.py

```diff
@@ -67,10 +67,7 @@ class Config(ConfigBase):
                        "cortex": cortex.Cortex.from_file(os.path.join(DEFAULT_SUBJECT_PATH,
                                                                       DEFAULT_CORT_SURFACE_ZIP),
                                                          region_mapping_file=os.path.join(DEFAULT_SUBJECT_PATH,
-                                                                                          DEFAULT_CORT_REGION_MAPPING_TXT),
-                                                         eeg_projection_file=
-                                                         os.path.join(DEFAULT_SUBJECT_PATH,
-                                                                      "QL_20120814_ProjectionMatrix.mat"))
+                                                                                          DEFAULT_CORT_REGION_MAPPING_TXT))
```

Comment:
I am only curious about why the eeg_projection_file is not needed here any more.

Reply:
This is because of the latest changes in tvb-library branch neotraits_py, in tvb/datatypes/cortex.py. We decided to simplify "from_file" method, because eeg_projection_file is not used in the project at this moment... but if you think you want to use eeg_projection_file in the future we can review this change.

Reply:
Indeed, the eeg_projection from class Cortex was not being used. Instead we need to set it on the Monitor object, if using EEG Projection Monitor